rclcpp/test/test_function_traits.cpp

```diff
@@ -15,6 +15,7 @@
 #include <gtest/gtest.h>
 
 #include <rclcpp/function_traits.hpp>
+#include <string>
```

Comment:
System headers like `string` should be included before user land header like `rclcpp`.